conans/test/unittests/client/build/meson_test.py

```diff
@@ -71,60 +71,66 @@ def folders_test(self):
             'includedir': 'include',
             'cpp_std': 'none'
         }
-
+        build_dir = os.path.join(self.tempdir, "build")
         meson.configure(source_dir=os.path.join(self.tempdir, "../subdir"),
-                        build_dir=os.path.join(self.tempdir, "build"))
+                        build_dir=build_dir)
         source_expected = os.path.join(self.tempdir, "../subdir")
-        build_expected = os.path.join(self.tempdir, "build")
-        cmd_expected = 'meson "%s" "%s" --backend=ninja %s --buildtype=release' \
-                       % (source_expected, build_expected, defs_to_string(defs))
+        build_expected = build_dir
+        cross_file = os.path.join(build_dir, "cross_file.txt")
+        cmd_expected = 'meson "%s" "%s" --backend=ninja %s --buildtype=release --cross-file=%s' \
+                       % (source_expected, build_expected, defs_to_string(defs), cross_file)
         self._check_commands(cmd_expected, conan_file.command)
 
-        meson.configure(build_dir=os.path.join(self.tempdir, "build"))
+        meson.configure(build_dir=build_dir)
         source_expected = os.path.join(self.tempdir, "my_cache_source_folder")
-        build_expected = os.path.join(self.tempdir, "build")
-        cmd_expected = 'meson "%s" "%s" --backend=ninja %s --buildtype=release' \
-                       % (source_expected, build_expected, defs_to_string(defs))
+        build_expected = build_dir
+        cmd_expected = 'meson "%s" "%s" --backend=ninja %s --buildtype=release --cross-file=%s' \
```

Comment:
As a general rule, new features in code shouldn't change existing tests, unless it is strictly considered a bug. Instead, please try to just add new tests for the new behavior.